docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -333,6 +333,7 @@ Return low-level information on the container `id`
 			"NetworkDisabled": false,
 			"OnBuild": null,
 			"OpenStdin": false,
+            "PortSpecs": null,
```

Comment:
yahi you're right! I spent 5 minutes looking for tabs but didn't look at previous line lol (and I don't highlight text in vim with mouse :/)